inject-groovy/src/test/groovy/io/micronaut/inject/requires/RequiresBeanPropertiesSpec.groovy

```diff
@@ -111,6 +116,11 @@ import jakarta.inject.Singleton
 class Config
 {
     Boolean boolProperty
+
+    // TODO: Why do we need this?  Groovy bug?
```

Comment:
Raised https://issues.apache.org/jira/browse/GROOVY-10762 to see if it's a bug

Reply:
It's expected and not a bug, fixed in https://github.com/micronaut-projects/micronaut-core/pull/8010/commits/fd8d88542740b94e192593203564274ca77f6874